start_admin.sh

```diff
@@ -228,6 +262,15 @@ for key_alg in rsa ecdsa ed25519; do
   fi
 done
 
+readonly host_cgroup="/.bottlerocket/rootfs/sys/fs/cgroup"
+
+cgroup_fstype=$(findmnt -n -o FSTYPE "${host_cgroup}" || true)
+if [[ "${cgroup_fstype}" == tmpfs ]]; then
```

Comment:
```suggestion
cgroup_fstype="$(findmnt -n -o FSTYPE "${host_cgroup}")"
if [[ "${cgroup_fstype}" == "tmpfs" ]]; then
```
Safer to wrap this in quotes. `|| true` also doesn't buy anything since you're validating it on the next line.

Reply:
Added in latest change